src/Microsoft.DotNet.Cli.Utils/CommandResolution/ProjectToolsPackageCommandResolver.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
```

Comment:
This and ProjectToolsCommandResolver differ by one line--- the declaration of s_toolPackageFramework. Is this a copy/paste error?